### PR TITLE
resources views root path replace bundle path

### PR DESCRIPTION
**Pull request by gonzaloalonsod**
$this->rootDir change $this->bundle->getPath()

### CODE REVIEW

**Comment by javiereguiluz** [comment]
@gonzakpo I'd say that `generate:crud` command is only used for application bundles (not third-party bundles). These app bundles store their templates in `app/Resources/views/` by default, so we should keep using `rootdir` instead of `bundle->getPath()`, right?

**Comment by gonzakpo** [comment]
Yes. Personally I prefer to have the views in the bundle, as it was before. Any chance of settings?
For example ask if bundle (not third-party bundles) would be acceptable?

File: Generator/DoctrineCrudGenerator.php [hunk 1]
```diff
@@ -78,7 +78,7 @@ public function generate(BundleInterface $bundle, $entity, ClassMetadataInfo $me
 
         $this->generateControllerClass($forceOverwrite);
 
-        $dir = sprintf('%s/Resources/views/%s', $this->rootDir, str_replace('\\', '/', strtolower($this->entity)));
+        $dir = sprintf('%s/Resources/views/%s', $this->bundle->getPath(), str_replace('\\', '/', strtolower($this->entity)));
```
Comment (omrqs) [inline Generator/DoctrineCrudGenerator.php]: I was suggested this:
$dir = sprintf('%s/Resources/views/%s', $this->bundle->getPath(), str_replace('\\', '/', ucfirst($this->entity)));

**Comment by xabbuh** [comment]
@javiereguiluz The `GenerateBundleCommand` has a `shared` option. Not sure if adding that was a good idea, but shouldn't the other command support the same for consistency?

**Comment by fabpot** [comment]
Closing this pull request as this bundle is now obsolete. Please, use the maker bundle instead.

**Pull request closed without merging**